docs/app/pages/components/components-sections/input-controls/input-controls.module.ts

```diff
@@ -68,7 +68,6 @@ const ROUTES = [
     ],
     exports: SECTIONS,
     declarations: [...SECTIONS, HighlightSearch],
-    entryComponents: SECTIONS
 })
 export class ComponentsInputControlsModule {
```

Comment:
I'm getting a console error when I navigate to this section. Im guessing we are missing a ReactiveFormsModule import:

![image](https://user-images.githubusercontent.com/20795331/75800217-1bf5c280-5d71-11ea-902d-2f31e1c7f1f5.png)

Reply:
Might be related, but the float label and number picker sections are missing:

![image](https://user-images.githubusercontent.com/20795331/75800298-3def4500-5d71-11ea-9d96-92adb38c2b05.png)

![image](https://user-images.githubusercontent.com/20795331/75800354-52cbd880-5d71-11ea-927b-16742c61f0f7.png)